scene/main/window.cpp

```diff
@@ -507,6 +507,7 @@ void Window::request_attention() {
 }
 
 void Window::move_to_foreground() {
```

Comment:
Please wrap the function and also its declaration in the header file in
``` c
#ifndef DISABLE_DEPRECATED
#endif // DISABLE_DEPRECATED 
```
as it was done for example in
https://github.com/godotengine/godot/blob/6916349697a4339216469e9bf5899b983d78db07/scene/2d/touch_screen_button.cpp#L372-L383

Reply:
Should something be done about the line below as well?
https://github.com/godotengine/godot/blob/6916349697a4339216469e9bf5899b983d78db07/scene/main/window.cpp#L2691

Reply:
Yes, that should also be enclosed in a DISABLE_DEPRECATED statement.